fix broken Accoutrement links

dvdherron opened the pull request:
![](https://source.unsplash.com/featured/?cute,animal&goo6e)
<!-- Swap `CHANGE_ME` for a random string to get a different image -->


<!--
## Description
Uncomment if you want to provide a custom description.
-->


## Steps to test/reproduce
See the list of Accoutrement modules at `/accoutrement/`
_Please explain how to best reproduce the issue and/or test the changes locally (including the pages/URLs/views/states to review)._


## Show me
_Provide screenshots/animated gifs/videos if necessary._

dvdherron commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Since we updated Accoutrement, we probably should remove the links and descriptions for Core and Init and add Tokens, Ratios, Utilities?](https://trello.com/c/oZDrk9MN/162-since-we-updated-accoutrement-we-probably-should-remove-the-links-and-descriptions-for-core-and-init-and-add-tokens-ratios-utili)

dvdherron commented:
I removed links and mentions to "Init" and "Core" on the Accoutrement page and added the "Tokens", "Ratios", and "Utilities" modules. 

Should we reuse the same icon (core) for these added modules? Or are there more of these icons to choose from someplace else? @stacyk @mirisuzanne 

![image](https://user-images.githubusercontent.com/41588129/159717077-6c185bc4-7d35-43fb-a8d1-586d7535075f.png)

jgerigmeyer reviewed (approved):
I notice that this list is slightly different than the list of modules on https://www.oddbird.net/accoutrement/docs/ but that might be intentional.

dvdherron commented:
> I notice that this list is slightly different than the list of modules on https://www.oddbird.net/accoutrement/docs/ but that might be intentional.

Hmm, yeah, these lists are slightly different. I added "Variables" to the list on OddSite. I'm guessing it would make sense to  add "Layout" to the list of "Modules for common data types" in the Accoutrement docs too  @mirisuzanne ?

 (Tokens is not in that list in the docs but is described in the following section)

mirisuzanne commented:
I think it would make most sense for both these lists to match the top-level headings in the Accoutrement docs sidebar navigation. With the possible special-casing of Tokens in the readme, as that's a meta-syntax that any of the other modules can rely on.

The icons are all from linearicons set, which we have in the company dropbox. I don't know if we still feel like there are icons that make sense for every module, or if we want to reconsider those. I think we either want to find a unique icon for each module, or reconsider those icons -- it's a bit strange to have only some of them repeated.

dvdherron commented:
> I think it would make most sense for both these lists to match the top-level headings in the Accoutrement docs sidebar navigation. With the possible special-casing of Tokens in the readme, as that's a meta-syntax that any of the other modules can rely on.

Makes sense. I edited this list to match the Accoutrement docs sidebar navigation. And I'll take care of the list on the docs page as well. 

> 
> The icons are all from linearicons set, which we have in the company dropbox. I don't know if we still feel like there are icons that make sense for every module, or if we want to reconsider those. I think we either want to find a unique icon for each module, or reconsider those icons -- it's a bit strange to have only some of them repeated.

Added new icons for Tokens and Ratios. @stacyk Can you tell why the outer circle path isn't showing up on the two icons I've added? Changing the viewbox to match the other icons makes the svg show up as a tiny pixels. 

![image](https://user-images.githubusercontent.com/41588129/159941057-489ff445-6406-4077-8e8d-1fdb9d01a585.png)

dvdherron commented on content/_includes/logos/accoutrement_tokens.svg:
```diff
@@ -0,0 +1,11 @@
+<svg
```
@mirisuzanne @stacyk 

Some other possibilities for "Tokens"

key-value pairs
  - key
  
![image](https://user-images.githubusercontent.com/41588129/159966205-5302e057-3643-4f21-ab7d-b473c8119cd7.png)

- keyhole

![image](https://user-images.githubusercontent.com/41588129/159966426-f9fed1c2-dd6f-43ff-a941-bf95902d669a.png)

Searching

- flashlight

![image](https://user-images.githubusercontent.com/41588129/159966724-828fe323-0637-47b9-9706-2dc0dd8df99c.png)

- binoculars

![image](https://user-images.githubusercontent.com/41588129/159967601-0f8f5f98-d9af-4d73-a626-1d1f0cc9887f.png)

Dictionary

- book

![image](https://user-images.githubusercontent.com/41588129/159968756-29c94406-716b-4b05-8617-819149325ebe.png)

mirisuzanne replied:
I kinda like the book.

stacyk replied:
And I'm team key. But I don't feel strongly about it

stacyk replied:
We could do something like building block-esque 

https://thenounproject.com/icon/modules-2209534/
![Screen Shot 2022-03-24 at 12 30 08 PM](https://user-images.githubusercontent.com/1581694/159975725-199a06db-0ee8-42b1-b4dc-6fba6ff5715a.jpg)
 
or tetris pieces
![Screen Shot 2022-03-24 at 12 31 45 PM](https://user-images.githubusercontent.com/1581694/159976026-4a676c43-e87c-4364-a92c-b266fb99aaa5.jpg)

https://thenounproject.com/icon/tetris-piece-689980/
https://thenounproject.com/icon/tetris-piece-3644812/

But I still like the simplicity of the key as well.

mirisuzanne replied:
I can get behind the key :)

dvdherron replied:
Key it is @mirisuzanne @stacyk 

Added the key icon in [e667273](https://github.com/oddbird/oddleventy/pull/201/commits/e6672730e913e0329addf6b0bf01360dbfe14f31)

![bild](https://user-images.githubusercontent.com/41588129/160113828-557f4a02-8003-44cb-a3aa-e7b41cdc8aec.png)

stacyk commented:
> Added new icons for Tokens and Ratios. @stacyk Can you tell why the outer circle path isn't showing up on the two icons I've added? Changing the viewbox to match the other icons makes the svg show up as a tiny pixels.
> 

@dvdherron The path for that circle is way outside the viewbox. I feel like we probably should have a stroke on these instead of a path, so they could all be consistent, but sometimes having more control over the elements is nice. I'll update the ratio icon for now but hold on that token one. 

![Screen Shot 2022-03-24 at 12 39 39 PM](https://user-images.githubusercontent.com/1581694/159977436-1d70261f-293b-425b-9b20-c6abdf48746e.jpg)

dvdherron commented:
> > Added new icons for Tokens and Ratios. @stacyk Can you tell why the outer circle path isn't showing up on the two icons I've added? Changing the viewbox to match the other icons makes the svg show up as a tiny pixels.
> 
> @dvdherron The path for that circle is way outside the viewbox. I feel like we probably should have a stroke on these instead of a path, so they could all be consistent, but sometimes having more control over the elements is nice. I'll update the ratio icon for now but hold on that token one.
> 
> ![Screen Shot 2022-03-24 at 12 39 39 PM](https://user-images.githubusercontent.com/1581694/159977436-1d70261f-293b-425b-9b20-c6abdf48746e.jpg)

Aha. Yeah, I noticed that and realized it needed to be resized and centered but couldn't get it to sit like the others. I think I'm not using the right tool to edit these. Maybe an SVG crash course? 

And as far as key versus book versus blocks. .  .I probably lean towards key or book more but I don't have a strong feeling here either @stacyk

stacyk commented:
@dvdherron I just pushed a fix. SVG is easiest to deal with in a program like Illustrator. You can see everything there, whether it is in frame or not. I'd be happy to do a walk through for anyone who'd like it.

stacyk reviewed (approved):
LGTM!
